src/modules/shared/components/Html/Html.tsx

```diff
@@ -26,7 +24,9 @@ const Html: FunctionComponent<HtmlProps> = ({
 			dangerouslySetInnerHTML={{
 				__html: DOMPurify.sanitize(content, sanitizePreset) as string,
 			}}
-			className={clsx(styles['c-html-wrapper'], className)}
+			// Avoid module scss file, since next serverside rendering can't add a style block to the document.
+			style={{ overflowWrap: 'break-word' }}
```

Comment:
Instead of defining an inline style; move this to the global .scss file

Reply:
fixed, moved to: src/styles/components/_html.scss